pnet_packet/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "pnet_packet"
+version = "0.19.0"
+authors = ["Robert Clipsham <robert@octarineparrot.com>"]
+license = "MIT/Apache-2.0"
+homepage = "https://github.com/libpnet/libpnet"
+repository = "https://github.com/libpnet/libpnet"
+description = "Cross-platform, binary packet parsing and manipulation"
+keywords = ["networking", "packet", "parsing", "raw"]
+categories = ["network-programming", "parser-implementations"]
+build = "build.rs"
+
+[features]
+default = ["with-syntex"]
+nightly = ["pnet_macros_plugin"]
+with-syntex = ["syntex", "pnet_macros/with-syntex"]
```

Comment:
By default we activate `with-syntex` and that in turn activates `pnet_macros/with-syntex`. But `pnet_macros` is an optional dependency and we don't activate that one by default. How does that work? If a feature of an optional dependency is activated, is the dependency also activated?

Reply:
My guess is it will activate `pnet_macros/with-syntex` when that optional dependency is used. I don't actually know though.

I'm thinking about scrapping the `with-syntex` stuff, and removing the `nightly` flag, since it hasn't worked for so long now, and I don't see that changing. It's probably easier than maintaining it. It could always be re-added later. What do you think?

Reply:
Sounds good. Anything simplifying the build system is a blessing at this point :P

Also, focusing on supporting stable Rust is the best I think.